pallets/liquid-staking-v2/src/lib.rs

```diff
@@ -100,6 +100,7 @@ mod pallet {
         type BaseXcmWeight: Get<Weight>;
 
         /// Account manages the staking assets.
+        #[pallet::constant]
```

Comment:
BaseXcmWeight also needs pallet::constant, can you correct also `pallet-liquid-staking`?

Reply:
maybe the new `DerivativeIndex` added by me needs also `constant` notation:)